### DISCUSSION TITLE
UI for PDF Editing

### DISCUSSION THREAD

**Discussion by rhemasystems** [Hopding/pdf-lib#1034] [General]
Thanks for the great work, but instead of writing the codes directly, is it possible to have a UI that will allow the same deatures like editing,commenting, and adding images and text to the PDF before downloading?

**Comment by rohit-daffodil** [comment]
WE are also looking for this. Any idea how to edit using some UI Like canvas ? 
We may need to load pdf on canvas and then from canvas we can drwa on pdf ?
